_posts/2018-08-01-mentor-spotlight-pablo-aguiar.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Mentor Spotlight: Pablo Aguiar"
+date: 2018-08-01 17:00:00
+categories: spotlights
+tags: HackIllinois2018, mentors, featured
+image: /assets/article_images/mentor-spotlight-pablo-aguiar/pablo.jpg
+author: Justin Yang
+---
+
+Welcome to our new series of mentor spotlights! In these interviews, we highlight some of the open source mentors that are central to HackIllinois.
+
+In this piece, we talked with Pablo Aguiar, who was a mentor at HackIllinois 2018. From the onset when we first approached him about mentoring at HackIllinois, Pablo's passion for open source has always shown through -- from our conversations leading up to the event to his working alongside students on open source projects at HackIllinois.
```

Comment:
- For tense continuity's sake, say "...we talk with Pablo Aguiar..."

- Second sentence of this is kind of a run-on. Plus you use the word "from" to start off both parts of it so it sounds a little awkward. reword this sentence - maybe split it into two sentences?